### PR TITLE
Issue 08: Made 'Popular Dishes' section in 'index.html' responsive.

### LINKED ISSUE #8: SeatAndEat : Make The Popular Dish Section Responsive
- Make the popular dish section responsive
- Also few changes to be done in the styling :
      * Make all the dish images card the same size
      * Make equal padding and same alignment to the cards

### PR DESCRIPTION
**Pull request by iamatifmoin**
## Linked Issue 

Closes: #8 

### Describe the changes you've made

1. Added class name (.card) to div elements in 'index.html' and removed the inline width of images.
2. Enhanced 'main.css' to make the 'Popular Dishes' section responsive.

## Type of change

What sort of change have you made:

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Describe how have you verified the changes made

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published in downstream modules.

## Screenshots

**On Larger Display:**

popular-dish-big

**On Smaller Display:**

popular-dish-small

### CODE REVIEW

**Review by Rupam-Matabber — COMMENTED** [review]
- Reduce the padding between the cards a little bit.
- Reduce the border radius a little bit.
- Try some absolute units for mentioning the height of the images. It will solve the issue. Make sure that the images are not distorted so much. A little amount of distortion is acceptable.
- make sure that the height and width of the images are the same.

@iamatifmoin Make this necessary changes.

**Comment by iamatifmoin** [comment]
@Rupam-Matabber I have made the following changes ->
1. The padding between cards has been reduced.
2. Border radius has been reduced.
3. Height and width of all images in now the same.

Note - The images are not distorted. The screenshots might be distorted.

**Comment by iamatifmoin** [comment]
Thank you for the tip for adjusting the height @Rupam-Matabber .

**Review by Rupam-Matabber — COMMENTED** [review]
- @iamatifmoin You forgot to change the border-radius in the media query.
- Make a breakpoint at 1200px width and make 2 grid columns.

Make these changes

**Comment by iamatifmoin** [comment]
@Rupam-Matabber I have made the suggested changes in the PR.

Note-> The border radius of the images gets inherited from above.

File: styles/main.css [hunk 1]
```diff
@@ -345,4 +345,84 @@ main .discount-card {
...
+@media only screen and (min-width:1201px) {
+  main .hero-menu-cards .container {
+    margin-top: 4rem;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    gap: 2rem;
+    text-align: center;
+  }
+  main .hero-menu-cards .container .card .different-curry {
+    width: 70%;
```
Comment (Rupam-Matabber) [inline styles/main.css]: Remove this width. It is making the image distorted above 1200px screens
image

**Review by Rupam-Matabber — COMMENTED** [review]
@iamatifmoin make this change and your pr will be ready to be merged.

**Comment by iamatifmoin** [comment]
> @iamatifmoin make this change and your pr will be ready to be merged.

@Rupam-Matabber Done!

**Review by Rupam-Matabber — COMMENTED** [review]
All good

**Comment by Rupam-Matabber** [comment]
Congrats @iamatifmoin your PR is merged.

**Comment by iamatifmoin** [comment]
> Congrats @iamatifmoin your PR is merged.

@Rupam-Matabber Thankyou!

**Pull request merged**